app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,11 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
+    return true if self[:child_id].nil?
+    role_allows?(:feature => self[:child_id])
```

Comment:
Yes, something like that @martinpovolny.  We are hoping to provide a better interface but to be honest, it's hard, we have the "what you can see"  (Rbac.search/get_view) separated from the "what can you do" (role_allows? / assert_privileges) so we'll have to figure out a way around that.